indexer/tasks/src/dsl/database_task.rs

```diff
@@ -12,38 +12,41 @@ pub struct BlockGlobalInfo {
     pub epoch_slot: Option<u64>,
 }
 
-pub type BlockInfo<'a, BlockType> = (
+pub type BlockInfo<'a, BlockType, BlockExtraType> = (
```

Comment:
what's this for? is it for supporting different blockchains?

is there a case where we would have the same `BlockType` but a different `BlockExtraType`?

Reply:
for every block type there can be just one block extra type